appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/admin/SetMetricsConfigurationCommand.java

```diff
@@ -85,11 +87,14 @@
     @Inject
     private Target targetUtil;
     
-    @Param(name = "enabled", optional = false)
+    @Param(name = "enabled", optional = true)
     private Boolean enabled;
     
-    @Param(name = "secureMetrics", optional = true, defaultValue = "false")
+    @Param(name = "secure", optional = true)
```

Comment:
This will reintroduce bug PAYARA-2661, please change the name of this param back